### DISCUSSION TITLE
New install, forgot admin credentials

### DISCUSSION THREAD

**Discussion by Johnster001** [alexjustesen/speedtest-tracker#1076] [Q&A]
I just set this up on my Synology using Portainer a week ago, and for some reason I forgot to record my admin credentials.  Is there a way to get them from the config files or am I looking at re-installing?

**Comment by Johnster001** [comment]
Any thoughts?  I can tear it down and redo but that feels like overkill.

**Reply by alexjustesen** [reply]
You can use the commands to reset an account's password from the CLI: https://docs.speedtest-tracker.dev/other/commands
